newdle/client/src/components/common/LanguageSelector.js

```diff
@@ -11,25 +11,19 @@ export default function LanguageSelector() {
   const language = useSelector(getLanguage);
   const dispatch = useDispatch();
 
-  if (window.location.hash !== '#i18n') {
```

Comment:
There's another check like this somewhere else. You need to remove that one as well or keep it here. I would prefer the latter, since we currently do not have any complete translations, so showing it would just add noise...